controllers/flux/minicluster.go

```diff
@@ -88,9 +88,10 @@ func (r *MiniClusterReconciler) ensureMiniCluster(
 	// Create headless service for the MiniCluster OR single service for the broker
 	selector := map[string]string{"job-name": cluster.Name}
 
-	// If we are adding a minimal service to the index 0 pod only
+	// If we are adding a minimal service expose the index 0 pod only
+	// LabelSelectors are ANDed
```

Comment:
@alculquicondor please confirm this, it will be good to have a double check from you here

Reply:
The docs can confirm for us: https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1LabelSelector.md

>  The requirements are ANDed.